.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="VarargParameter" enabled="false" level="INFORMATION" enabled_by_default="false" />
```

Comment:
You sure?

Reply:
> Reports methods taking a variable number of arguments, also known as varargs methods. Such methods are not supported under Java 1.4 or earlier JVMs. The quickfix of this inspection replaces a variable argument parameter with the equivalent array parameter. Relevant arguments in calls to the method are wrapped in an array initializer expression. 

We don't have to worry about compat with Java 1.4, so I think the only other reason to have it enabled is so that quickfix-action of replace-variable-argument-parameter-with-array shows up when you have the cursor on varargs.

we can keep it at default if that's useful, but so far I've found it to be an unhelpful suggestion.